test/dataform/test_sparse.py

```diff
@@ -0,0 +1,61 @@
+'''Tests for dataform'''
+
+
+import unittest
+
+import tempfile
+
+import numpy as np
+
+from bdpy.dataform.sparse import load_array, save_array
+
+
+class TestUtil(unittest.TestCase):
+
+    def test_load_save_dense_array(self):
+        payloads = [
+            [(10,), 'test_array_dense_ndim1.mat'],  # ndim = 1
+            [(3, 2), 'test_array_dense_ndim2.mat'],  # ndim = 2
+            [(4, 3, 2), 'test_array_dense_ndim3.mat']  # ndim = 3
+        ]
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            for shape, fname in payloads:
+                original_data = np.random.rand(*shape)
+                save_array(tmpdir + '/' + fname, original_data, key='testdata')
+                from_file = load_array(tmpdir + '/' + fname, key='testdata')
+
+                np.testing.assert_array_equal(original_data, from_file)
+
+    def test_load_save_sparse_array(self):
+        payloads = [
+            [(10,), 'test_array_sparse_ndim1.mat'],  # ndim = 1
+            [(3, 2), 'test_array_sparse_ndim2.mat'],  # ndim = 2
+            [(4, 3, 2), 'test_array_sparse_ndim3.mat']  # ndim = 3
+        ]
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            for shape, fname in payloads:
+                original_data = np.random.rand(*shape)
+                original_data[original_data < 0.8] = 0
+
+                save_array(tmpdir + '/' + fname, original_data, key='testdata', sparse=True)
+                from_file = load_array(tmpdir + '/' + fname, key='testdata')
+
+                np.testing.assert_array_equal(original_data, from_file)
+
+    # TODO: test/data/{<file>.mat} is not included in the repository
```

Comment:
I found the test code that assumes the existence of some mat files, which are not included in this repository. Do you know what type of dataset we need? > @ShuntaroAoki

Reply:
ref. to the original code:
https://github.com/KamitaniLab/bdpy/blob/4e87ee15ec6991c040a85e297cde0507ee54b1b0/test/test_dataform_sparse.py#L65-L74